content/src/content/jcr_root/apps/core/wcm/components/modal/v1/modal/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:description="Modal Dialog"
+    jcr:primaryType="nt:unstructured"
+    jcr:title="Modal"
+    sling:resourceType="cq/gui/components/authoring/dialog"
+    helpPath="https://www.adobe.com/go/aem_cmp_modal_v1"
+    trackingFeature="core-components:modal:v1">
+    <content
+        jcr:primaryType="nt:unstructured"
+        sling:resourceType="granite/ui/components/coral/foundation/container">
+        <items jcr:primaryType="nt:unstructured">
+            <fixedcolumns
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns">
+                <items jcr:primaryType="nt:unstructured">
+                    <properties
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured">
+                            <modal-id
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                disabled="{Boolean}true"
+                                fieldLabel="Modal Id"
+                                name="./modalId"/>
+                            <description
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                fieldDescription="Provide description for the modal"

```

Comment:
I think we can come up with some better descriptions for these fields that add useful information about the intention of the input, rather than repeating the label [0].

Example:

**Instead of**
```Add a title for the modal```

**Prefer**
```A title that is displayed in the modal header```

[0] - https://github.com/adobe/aem-core-wcm-components/wiki/Dialog-Style-Guide#field-description